tests/testthat/test-tokens.R

```diff
@@ -420,23 +420,23 @@ test_that("tokens works as expected with NA, and blanks", {
     )   
     expect_equal(
         as.list(tokens(c("one", NA, ""))),
-        list(text1 = "one", text2 = "NA", text3 = character())
```

Comment:
Wow - we actually tested the error condition as if we expected it! This was originally clearly not a good test.